mibidata/segmentation.py

```diff
@@ -75,11 +86,59 @@ def _quadrant_mean(inds, image):
     upper_left = vals[(inds[0] <= y_center) & (inds[1] < x_center)].sum(axis=0)
     upper_right = vals[(inds[0] < y_center) & (inds[1] >= x_center)].sum(axis=0)
     lower_left = vals[(inds[0] > y_center) & (inds[1] <= x_center)].sum(axis=0)
-    lower_right = vals[(inds[0] >= y_center) & (inds[1] > x_center)].sum(axis=0)
+    lower_right = vals[((inds[0] >= y_center) & (inds[1] > x_center))
+                       | ((inds[0] == y_center) & (inds[1] == x_center))] \
+                       .sum(axis=0)
     quads = np.stack((upper_left, upper_right, lower_left, lower_right), axis=1)
+    # sanity check: test that all pixels in the cell are used
+    assert_array_equal(vals.sum(axis=0), quads.sum(axis=1))
```

Comment:
I think you can remove this now from the final code now that you've tested it.

Reply:
done